src/style.css

```diff
@@ -1,3 +1,87 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #fff;
+  color: #00acdc;
+}
+
 body {
   margin: 0;
+  height: 60px;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.moyo_logo {
+  margin-top: 10px;
+  padding-left: 10px 0;
+  display: flex;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: flex-end;
+}
```

Comment:
1. Move those styles to the top and reset them for all <ul>. Do the same for all reseting styles
```
ul {
  list-style: none;
  margin: 0;
  padding: 0;
}
```
2. `position: relative;` is redundant here